src/lecturer/components/createQuestionView/CreateQuestionView.tsx

```diff
@@ -24,10 +24,42 @@ export function CreateQuestionView() {
     const theme = useTheme();
     const store = useContext(StoreContext);
 
-    const [title, setTitle] = useState<string>("");
-    const [question, setQuestion] = useState<string>("");
-    const [inputList, setInputList] = useState<string[]>([]);
-    const [checked, setChecked] = useState<boolean[]>([]);
+
+    enum QuestionType {
+        CLOSED,
+        OPEN
+    }
+
+    let titleVal = "";
+    let questionVal = "";
+    let modeVal = QuestionType.CLOSED;
+    let answersVal: string[] = [];
+    let isCorrectVal: boolean[] = [];
+
+    if (store.selectedQuestion != -1) {
```

Comment:
I don't think that storing selected question globally is necessary, is it used in other components? If not maybe pass it in props.

Reply:
The only thing it could enable is the option of changing tabs and returning to the edited question. But we don't have an edit question tab.